.github/workflows/pkgci_regression_test.yml

```diff
@@ -139,7 +139,9 @@ jobs:
           # CPU
           - name: cpu_llvm_task
             models-config-file: models_cpu_llvm_task.json
-            runs-on: nodai-amdgpu-w7900-x86-64
+            runs-on:
+              - self-hosted # must come first
+              - persistent-cache
```

Comment:
I added this new `persistent-cache` label to the mi250 and mi300 runners, as well as the w7900 runners.

Not sure if this will work on the mi300 though, since it skips `Pre Checkout MI300 Step`: https://github.com/iree-org/iree/blob/5b112cbc21046aaa028a05da62da1e3f8d19bc20/.github/workflows/pkgci_regression_test.yml#L172-L177